File: PlatziCodingChallenge_Dia23/InteresCompuesto2.py
#¿Puedo ahorrar otros periodos de tiempo? en Python

def finalAmount(capital, time, rate):
    return capital * (1 + rate)**time # C = c(1 + r)^t

#Pedido de datos
capital = 1000
tiempo = int(input('¿Por cuantos meses vas a ahorrar?: '))
interes = 4/100 # 4%
print('En {} mes(es) vas a tener ${}'.format(tiempo, round(finalAmount(capital, tiempo, interes), 2)))

File: Dia23/InteresCompuesto2.js
//¿Puedo ahorrar otros periodos de tiempo? en JavaScript

function finalAmount(capital, time, rate) {
    return capital * Math.pow(1 + rate, time) //C = c(1 + r)^t
}

//Pedido de datos
const capital = 1000
const tiempo = parseInt(prompt('¿Por cuantos meses vas a ahorrar?'))
const interes = 4/100 //4%
alert(`En ${tiempo} mes(es) vas a ahorrar $${finalAmount(capital, tiempo, interes).toFixed(2)}`)